scipy/special/tests/test_pdtr.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import scipy.special as sc
+import pytest
+from numpy.testing import assert_almost_equal, assert_array_equal
+from numpy import isnan
```

Comment:
Let's write out `np.isnan` and avoid this import.

Reply:
done